tests/scripts/t3000/run_t3000_unit_tests.sh

```diff
@@ -102,8 +117,12 @@ run_t3000_tests() {
 
   # Run mixtral tests
   run_t3000_mixtral_tests
+
+  # Run ttmetal tests
```

Comment:
why we move this test?
out of curiosity

Reply:
reverted